src/transports/dash/segment_parser.ts

```diff
@@ -40,6 +41,12 @@ export default function parser({ content,
   const { period, representation, segment } = content;
   const { data, isChunked } = response;
   if (data == null) {
+    const isStaticContent = segment.isInit && segment.range === undefined;
```

Comment:
What if we're on a SegmentTemplate or SegmentTimeline?

Reply:
Done